src/Forms/__stories__/FormGroup.stories.js

```diff
@@ -1,7 +1,7 @@
 import FormGroup from '../FormGroup';
 import FormItem from '../../Forms/FormItem';
 import FormLabel from '../../Forms/FormLabel';
```

Comment:
Minor nit:  both `FormItem` and `FormLabel` could have paths simplified to `../FormItem` or `../FormLabel`, respectively.